src/pages/planters/[planterid].js

```diff
@@ -233,9 +233,15 @@ export default function Planter(props) {
               <Box sx={{ mt: 2 }}>
                 <Info
                   iconURI={CalendarIcon}
-                  info={`Planter since ${moment(planter.created_at).format(
-                    'MMMM DD, YYYY',
-                  )}`}
+                  info={
+                    <>
+                      Planter since
+                      <time dateTime={planter?.created_at}>
+                        &nbsp;
```

Comment:
Can we move this outside the `time` element, in React we can use something like `{" "}` to create a extra space after **Planter since**